mdn-drafts/isExtended.md

```diff
@@ -0,0 +1,35 @@
+---
+recipe: api-interface
+title: 'Screen.isExtended'
```

Comment:
Should we also add something like this for Screen.onchange?

Reply:
Yes. I'll do it in a separate PR. Let's get this one wrapped.

Reply:
Ack.